app/helpers/home_helper.rb

```diff
@@ -16,4 +16,12 @@ def contact_text
       concat '.'
     end
   end
+
+  def header_path
+    File.join('labels', Whitelabel[:label_id], 'header.png')
+  end
+
+  def header?
```

Comment:
IMHO this method name is to generic and doesn't communicate the intend. How about `header_image_available?`, `uses_header_image?`